app/models/car.rb

```diff
@@ -1,5 +1,5 @@
 class Car < ActiveRecord::Base
-  has_attached_file :image, styles: { medium: '300x300>', thumb: '100x100>' }, default_url: 'shredding_brah2.png'
+  has_attached_file :image, styles: { medium: '200x200>', thumb: '100x100>' }, default_url: 'car_default.png'
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.